sys/meson.py

```diff
@@ -172,6 +172,11 @@ def win_dist(args):
     with open(r2_bat_fname, 'w') as r2_bat:
         r2_bat.write('@"%~dp0\\radare2" %*\n')
 
+    r2_sh_fname = args.install + r'\bin\r2'
```

Comment:
uhm isnt this meson.py suposed to run on windows too?

Reply:
it does ... this pr is for people running POSIX shells on Windows (e.g. the bash shell that comes with [MSYS2](https://www.msys2.org/))